docs/system/DATA_MODELS.md

```diff
@@ -88,6 +88,7 @@
         "country" : "<country of journal publication>",
         "license" : [<list of titles of licences>],
         "publisher" : "<publisher>",
+        "institution" : "<institution>",
```

Comment:
should we have these doc changes in the DataObj docs? docs/system/Journal.md and docs/system/Suggestion.md

Reply:
In addition, it looks here like you've added institution to the index of Article, but in the DataObj code below it's been added to bibjson.journal

Reply:
It should be in bibjson.journal as it is already implemented there. Actually it also should be added to Article.index.institution to make this pr work correctly. And therefore the migration should be run on the articles as well.